strings/ion_laws.json

```diff
@@ -87,9 +87,6 @@
         "INVISIBLE",
         "PAINFUL",
         "HARMFUL",
-        "HOMOSEXUAL",
-        "HETEROSEXUAL",
-        "SEXUAL",
```

Comment:
For reference, these aren't quite as explicit as the others; this would primarily happen like this:

"The shuttle cannot be called because of one hundred homosexual chaplains on the station."